README.md

```diff
@@ -1,4 +1,4 @@
-Sideboard [![Build Status](https://travis-ci.org/appliedsec/sideboard.svg)](https://travis-ci.org/appliedsec/sideboard)[![Coverage Status](https://coveralls.io/repos/appliedsec/sideboard/badge.png)](https://coveralls.io/r/appliedsec/sideboard)
+Sideboard [![Build Status](https://travis-ci.org/magfest/sideboard.svg)](https://travis-ci.org/magfest/sideboard)[![Coverage Status](https://coveralls.io/repos/magfest/sideboard/badge.png)](https://coveralls.io/r/magfest/sideboard)
```

Comment:
The doc links date back to when this was originally open sourced, before my previous employer took it offline (and implicitly made MAGFest the canonical owner of the project).